packages/hoppscotch-backend/src/shortcode/shortcode.service.ts

```diff
@@ -104,35 +112,44 @@ export class ShortcodeService implements UserDataHandler, OnModuleInit {
    * Create a new ShortCode
    *
    * @param request JSON string of request details
-   * @param userUID user UID, if present
+   * @param userInfo user UI
+   * @param properties JSON string of embed properties, if present
    * @returns Either of ShortCode or error
    */
-  async createShortcode(request: string, userUID: string | null) {
-    const shortcodeData = stringToJson(request);
-    if (E.isLeft(shortcodeData)) return E.left(SHORTCODE_INVALID_JSON);
+  async createShortcode(
```

Comment:
If we pass `"null"` stringify null value in the properties, should we treat as invalid input?

![Screenshot 2023-11-07 at 11 17 37 AM](https://github.com/hoppscotch/hoppscotch/assets/30154643/6aeebe64-d9b0-475c-9775-7dc948e21fc5)